src/webrtc/peerconnection.ts

```diff
@@ -629,6 +656,96 @@ export class PeerConnectionClass implements PeerConnection<signals.Message> {
   }
 }  // class PeerConnectionClass
 
+// Takes a DataChannel and provides a ControlChannel API atop of it.
+class ControlChannelWrapper implements datachannel.ControlChannel {
```

Comment:
I'll add that.  While I'm at it, about half of handleMessage_ is debug-logging-related.  Want me to take that out to shorten this?